DevelopedCalculator.js

```diff
@@ -1,5 +1,5 @@
 class DevelopedCalculator extends Calculator {
-    history = [];
+    history = this.currentResult;
```

Comment:
why did you change this to `this.currentResult`?

Reply:
@SMH110 SMH110 4 hours ago 
it needs to be array to hold more than on value

every time, a button is clicked, we need to push new value to this array

the new value is this.currentResult

Reply:
Here you define the array, in the method which will be called when the buttons are clicked you push the current result to this array